### PR TITLE
Mental Health Consultations

### PR DESCRIPTION
**Pull request by floresj**
Issues: #803, #833, #803
Bugs: #834, #830, #831
- Added Consultation component
- Renamed `CompetenceItemValidator` to `OrderValidator`

### CODE REVIEW

**Comment by floresj** [comment]
Ok, I just made a couple more updates. This should be good now.

File: src/config/locales/en.js [hunk 1]
```diff
@@ -3972,6 +3974,10 @@ const en = {
           title: 'Need help with whether this matter was appealed?',
           message: '',
```
Comment (bmallred) [inline src/config/locales/en.js]: Some of these help messages are missing actual `message` verbiage

**Comment by floresj** [comment]
Updated the help message content. Also included some bug fixes based on Jay's feedback.

File: src/components/Section/Psychological/Order.jsx [hunk 2]
```diff
@@ -137,6 +137,18 @@ export default class Order extends ValidationElement {
...
+
+        <div className="eapp-field-wrap no-label">
+          <h3>{i18n.t(`psychological.${prefix}.heading.disposition`)}</h3>
+          <Help id={`psychological.${prefix}.help.disposition`}>
+            <Text name="Disposition"
+              className="disposition"
+              bind={true}
+              onValidate={this.props.onValidate}
+            />
+            <HelpIcon className="date-help-icon" />
```
Comment (bmallred) [inline src/components/Section/Psychological/Order.jsx]: Is `date-help-icon` the right class name?

File: src/components/Section/Psychological/Psychological.jsx [hunk 3]
```diff
@@ -93,19 +93,19 @@ class Psychological extends ValidationElement {
 
           <SectionView name="competence"
             back="psychological/intro"
-            backLabel={ i18n.m('psychological.destination.intro') }
+            backLabel={ i18n.t('psychological.destination.intro') }
             next="psychological/consultations"
-            nextLabel={ i18n.m('psychological.destination.consultation') }>
+            nextLabel={ i18n.t('psychological.destination.consultation') }>
```
Comment (bmallred) [inline src/components/Section/Psychological/Psychological.jsx]: Good catch... didn't even notice this last time.

File: src/components/Section/Psychological/Order.jsx (hunk 2 above)
Comment (floresj) [inline src/components/Section/Psychological/Order.jsx]: Doh! Good catch. Will change.

**Pull request merged**